packages/compass-web/src/connection-storage.tsx

```diff
@@ -96,6 +110,49 @@ function getMetricsIdAndType(
   };
 }
 
+// At the time of writing these are the possible instance sizes. If we main the
```

Comment:
Does it show in vscode on calling sites if is not a jsdoc? `/**`

Maybe is enough to show a few examples and link to some external documentation or source?

Reply:
I'll just remove it and replace it with a link, I think. That was mostly for draft PR review if anything.

Reply:
I also moved it to the type AtlasClusterMetadata where instanceSize is now specified.